nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeo.conf

```diff
@@ -7,11 +7,10 @@
 ##-----------------------------------------------------------------------------
 ## Java
 ##-----------------------------------------------------------------------------
-#JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
-#JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_152.jdk/Contents/Home
-
-## Heap size
-JAVA_OPTS=-Xms512m -Xmx1024m
+## You can control the JVM heap size with the following options: InitialRAMPercentage and MaxRAMPercentage.
+## They are respectively equivalent to Xms and Xmx, using a percentage of the cgroup memory limit.
```

Comment:
.. using a percentage of the available memory (or cgroup memory limit when running inside a container)